18F/identity-idp
# Revise IdV app alert content to static content

**Why**: Avoid extra complexity around dealing with edge cases with alerts (e.g. warning subpages like with "Forgot Password" screen).

The previous implementation was also done in anticipation that we may need to omit status messages from step reuse such as personal key reset via account dashboard. We can sort through this when the time comes, but it's likely we could control this with either context or a prop of the step component.

**Testing Instructions:**

You will need to set the step as enabled in your local `config/application.yml`:

```yml
development:
  idv_api_enabled_steps: '["password_confirm","personal_key","personal_key_confirm"]'
```

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing flow up to password confirmation step
5. Observe success alert "We found records matching your Phone of record"
6. Click "Follow these instructions" (adjacent "Forgot your password?")
7. Observe that the success alert is no longer visible

**Screenshot:**

Before|After
---|---
![Screen Shot 2022-05-18 at 8 55 27 AM](https://user-images.githubusercontent.com/1779930/169044030-f3ca5823-e6c3-40c3-9be2-0fa49775b817.png)|![Screen Shot 2022-05-18 at 8 55 41 AM](https://user-images.githubusercontent.com/1779930/169044033-748f940d-0a25-45b2-8683-6726cea64eb0.png)
